docs/source/integrations/integration-index.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Apollo Server Integrations
+---
+
+Apollo Server 4 includes two built-in integrations: [`startStandaloneServer`](../api/standalone) and [`expressMiddleware`](../api/express-middleware).
+
+The `startStandaloneServer` function sets useful defaults to get you started quickly. Under the hood, the `startStandaloneServer` function uses Apollo Server 4's Express integration (i.e., `expressMiddleware`). The `expressMiddleware` function is Apollo Server 4's [Express](https://expressjs.com/) integration.
+
+> Are you looking to build a new integration? Or help maintain an existing integration? See [Building Web Framework Integrations for Apollo Server](./building-integrations) for step-by-step guidance!
+
+Apollo and the larger community maintain the following open-source integrations for Apollo Server:
+
+<!-- prettier-ignore -->
+| Web Framework | Integration Package |
+|--------|-------------|
+| AWS Lambda | [`@as-integrations/aws-lambda`](https://www.npmjs.com/package/@as-integrations/aws-lambda) |
```

Comment:
Let's make sure this has a reasonable README soon! cc @michael-watson @trevor-scheer